### DISCUSSION TITLE
Please support android 11 storage access

### DISCUSSION THREAD

**Discussion by developer-rojas** [tyron12233/CodeAssist#473] [Ideas]
hello, I have a device with Android 11, I find it a bit annoying to have my project saved in the android/data/com.tyron.code/projects folder, I would like to be able to create it in a different folder than the one I can choose, but it does not have compatibility with android 11. please can you add this tag in the android manifest file inside the "<application" tag

`android:requestLegacyExternalStorage="true"
`

thanks in advance, together we reach more

**Comment by IrnkvezzDeveloper** [comment]
you know that in the code too permission should be requested? smart guy who read Google and gives advice to more experienced coders
